Library/Homebrew/rubocops/cask/url.rb

```diff
@@ -0,0 +1,37 @@
+# typed: true
+# frozen_string_literal: true
+
+require "forwardable"
```

Comment:
Not sure what this does, I copied and pasted from other cops. 😳

Reply:
I think it's meaningless without any use of `def_delegator(s)`: https://ruby-doc.org/stdlib-2.5.1/libdoc/forwardable/rdoc/Forwardable.html (which would make a nice cop by itself)

Reply:
I thought so, but it's in a few cops that don't make use of `def_delegators` either. I guess that's a mistake. One RuboCop at a time for me, though. 😆